README.md

```diff
@@ -77,34 +26,43 @@ Usage:
-  logs         Get the logs from your running services.
-  restart      Restart the local development environment for a site.
-  remove       Remove an application's local services.
-  sequelpro    Easily connect local site to sequelpro
-  ssh          SSH to an app container.
-  start        Start the local development environment for a site.
-  stop         Stop an application's local services.
-  version      print ddev version and component versions
+  auth-pantheon Provide a machine token for the global pantheon auth.
+  config        Create or modify a ddev application config in the current directory
+  describe      Get a detailed description of a running ddev site.
+  exec          Execute a shell command in the container for a service. Uses the web service by default.
+  help          Help about any command
+  hostname      Manage your hostfile entries.
+  import-db     Import the database of an existing site to the local dev environment.
+  import-files  Import the uploaded files directory of an existing site to the default public upload directory of your application.
+  list          List applications that exist locally
+  logs          Get the logs from your running services.
+  pull          Import files and database using a configured provider plugin.
+  remove        Remove the local development environment for a site.
+  restart       Restart the local development environment for a site.
+  sequelpro     Easily connect local site to sequelpro
+  ssh           Starts a shell session in the container for a service. Uses web service by default.
+  start         Start the local development environment for a site.
+  stop          Stop the local development environment for a site.
+  version       print ddev version and component versions
+
+Flags:
+  -h, --help          help for ddev
+  -j, --json-output   If true, user-oriented output will be in JSON format.
 
 Use "ddev [command] --help" for more information about a command.
 ```
 
-## Getting Started - Documentation
-Once you've installed ddev, check out the [ddev Documentation Site](https://ddev.readthedocs.io) for information on how to get started and how to use ddev.
-
 ## Support
 If you've encountered trouble using ddev, please use these resources to get help with your issue:
 
 1. Please review the [ddev Documentation](https://ddev.readthedocs.io) to ensure your question isn't answered there.
 2. Review the [ddev issue queue](https://github.com/drud/ddev/issues) to see if an issue similar to yours already exists.
 3. If you've exhausted these options and still need help, please [file an issue](https://github.com/drud/ddev/issues/new) following the pre-populated guidelines and our [Contributing Guidelines](https://github.com/drud/ddev/blob/master/CONTRIBUTING.md) as best as possible.
+4. We also have a channel (#ddev) in the [Drupal Slack](https://www.drupal.org/slack) account. We're fairly responsive to questions and requests during normal business hours (US Mountain Time).
```

Comment:
"We try to be very responsive, but replies may lag outside normal business hours in US Mountain Time"

Reply:
+1